pkg-kni/nrtstatus/nrtstatus.go

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nrtstatus
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/k8stopologyawareschedwg/podfingerprint"
+)
+
+const (
+	BaseDirectory = "/run/nrtstatus"
```

Comment:
We might want to mount this directory on the host in case the pod is crashing.

Reply:
I'm not sure. It's something we will need to evaluate, but can be postponed for later I guess